packages/app-content-pages/src/screens/Publications/components/Project/Project.js

```diff
@@ -24,10 +25,57 @@ const Placeholder = ({ title }) => {
   )
 }
 
-function Project(props) {
-  const { avatarSrc, id, title, publications } = props
+const DEFAULT_HANDLER = () => true
+const DEFAULT_PUBLICATIONS = []
+
+function slugify(string) {
+  return string
+    // convert letters to lowercase
+    .toLowerCase()
+    // replace any spaces with Z
+    .replace(/\s/g, 'Z')
+    // remove all punctuation and whitespace
+    .replace(/\W/g, '')
+    // replace any Zs with a single dash
+    .replace(/Z+/g, '-')
```

Comment:
Why do the step with `Z` rather than `.toLowerCase().replaceAll(' ', '-')`?

Reply:
I also wanted to avoid `--` appearing in slugs eg. `backyard-worlds--planet-9` and this was the first solution that came to mind.